services/sonar/sonar-redirector.service.js

```diff
@@ -26,7 +26,7 @@ module.exports = [
       base: 'sonar',
       pattern: ':protocol(http|https)/:host(.+)/:component(.+)/:metric',
     },
-    transformPath: ({ component, metric }) => `/sonar/${component}/${metric}`,
+    transformPath: ({ component, metric }) => `/sonar/${metric}/${component}`,
```

Comment:
To avoid redirecting twice, we should make this same change to the above redirector too (line 15)